src/components/PhotoUpload/PhotoUploadButton.js

```diff
@@ -40,12 +41,12 @@ class PhotoUploadButton extends React.Component<Props, State> {
       <button 
         onClick={this.onClick} 
         className={`${className} link-button`}
-        aria-label={t`Upload images`} >
+        aria-label={t`Add images`} >
         <span className='button-icon'>
             <CameraIcon />
         </span>
-        <span className='button-label'>{t`Upload images`}</span>
-        <span className='motivation-hint'> {t`Your Good Deed!`}</span>
+        <span className='button-label'>{t`Add images`}</span>
+        <IncentiveHint>{t`Improve your karma!`}</IncentiveHint>
```

Comment:
Text changed

Reply:
Intentionally, this is the original string. There are existing translations for this incl. English where the translated string is named 'your good deed!'. Silke and Svenja will work on English wording in the transifex interface so at some point we need a process to replace 'original' strings with their changes.